saddle-core/src/main/scala/org/saddle/scalar/ScalarTagChar.scala

```diff
@@ -14,14 +14,28 @@
   */
 package org.saddle.scalar
 
-import org.saddle.ORD
+import org.saddle.{ORD, NUM, Index, Vec}
 import org.saddle.array.Sorter
+import org.saddle.locator.{Locator, LocatorAny}
+import org.saddle.index.{IndexAny}
 
 /** Char ScalarTag
   */
-object ScalarTagChar extends ScalarTagAny[Char] {
-  override def parse(s: String) = s.head
-  override def makeSorter(implicit ord: ORD[Char]): Sorter[Char] =
+object ScalarTagChar extends ScalarTag[Char] {
+  def parse(s: String) = s.head
+  def makeSorter(implicit ord: ORD[Char]): Sorter[Char] =
     Sorter.charSorter
-  override def missing: Char = Char.MinValue
+  def missing: Char = Char.MinValue
+  def isMissing(t: Char): Boolean = t == missing
+
+  def inf(implicit ev: NUM[Char]): Char = sys.error("inf not supported")
```

Comment:
Feels weird to have this here. Belongs to the Numeric typeclass imo.

Reply:
I agree